### PR TITLE
Simplify the experimental env var handling.

### PR DESCRIPTION
**Pull request by dlorenc**
Ref #67

### CODE REVIEW

File: cmd/cosign/cli/sign.go [hunk 1]
```diff
@@ -69,13 +69,13 @@ func Sign() *ffcli.Command {
 		kmsVal      = flagset.String("kms", "", "sign via a private key stored in a KMS")
 		upload      = flagset.Bool("upload", true, "whether to upload the signature")
 		payloadPath = flagset.String("payload", "", "path to a payload file to use rather than generating one.")
-		forceTlog   = flagset.Bool("force-tlog", false, "whether to upload to the tlog even when the image is private")
+		force       = flagset.Bool("-f", false, "skip warnings and confirmations")
```
Comment (font) [inline cmd/cosign/cli/sign.go:72]: Is requiring two dashes intentional here?

Comment (dlorenc) [reply cmd/cosign/cli/sign.go:72]: d'oh. nope!

File: cmd/cosign/cli/sign.go [hunk 2]
```diff
@@ -69,13 +69,13 @@ func Sign() *ffcli.Command {
...
 		annotations = annotationsMap{}
 	)
 	flagset.Var(&annotations, "a", "extra key=value pairs to sign")
 	return &ffcli.Command{
 		Name:       "sign",
-		ShortUsage: "cosign sign -key <key> [-payload <path>] [-a key=value] [-upload=true|false] [-force-tlog=true|false] <image uri>",
+		ShortUsage: "cosign sign -key <key> [-payload <path>] [-a key=value] [-upload=true|false] <image uri>",
```
[the remark this answers is not in the document]
Comment (dlorenc) [reply cmd/cosign/cli/sign.go]: Done!

**Review by priyawadhwa — APPROVED** [review]
💯

**Pull request merged**